[2023-06-21 14:33:23] [WARNING]: 虚拟设备信息不存在, 将自动生成随机设备. 
[2023-06-21 14:33:30] [WARNING]: 登录需要滑条验证码, 请验证后重试. 
[2023-06-21 14:33:30] [WARNING]: 请选择提交滑块ticket方式: 
[2023-06-21 14:33:30] [WARNING]: 1. 自动提交 
[2023-06-21 14:33:30] [WARNING]: 2. 手动抓取提交 
[2023-06-21 14:33:30] [WARNING]: 请输入(1 - 2)： 
[2023-06-21 14:33:36] [WARNING]: 请前往该地址验证 -> https://captcha.go-cqhttp.org/captcha?id=0MzwgmUD&style=simple&aid=16&uin=2470751924&sid=1054984500914854330&cap_cd=R4HL2P3v-gjeexGMZHHqK1q7EXokqD0085oszzVdWBq1mTqpxE4j3w**&clientype=1&apptype=2  
[2023-06-21 14:34:11] [WARNING]: 登录失败: 登录失败，建议升级最新版本后重试，或通过问题反馈与我们联系。 Code: 45 
[2023-06-21 14:34:11] [WARNING]: 你的账号涉嫌违规被限制在非常用设备登录, 请在手机QQ登录并根据提示完成认证 
[2023-06-21 14:34:11] [WARNING]: 或使用 -update-protocol 升级到最新协议后重试 
[2023-06-21 14:35:53] [ERROR]: 获取T544时出现问题: Post "https://captcha.go-cqhttp.org/sdk/dandelion/energy": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 
[2023-06-21 14:35:53] [WARNING]: 登录需要滑条验证码, 请验证后重试. 
[2023-06-21 14:35:53] [WARNING]: 请选择提交滑块ticket方式: 
[2023-06-21 14:35:53] [WARNING]: 1. 自动提交 
[2023-06-21 14:35:53] [WARNING]: 2. 手动抓取提交 
[2023-06-21 14:35:53] [WARNING]: 请输入(1 - 2)： 
[2023-06-21 14:35:57] [WARNING]: 请前往该地址验证 -> https://captcha.go-cqhttp.org/captcha?id=vw5xG8Gd&style=simple&aid=16&uin=2470751924&sid=3282246019681786441&cap_cd=EjmaMlxAkAi5nln-0_pEZR1xjneVY7qwpAJCrjcuPwAdxt_34km5cg**&clientype=1&apptype=2  
[2023-06-21 14:36:19] [ERROR]: 获取T544时出现问题: Post "https://captcha.go-cqhttp.org/sdk/dandelion/energy": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 
[2023-06-21 14:36:20] [WARNING]: 登录失败: 当前QQ版本过低，请升级至最新版本后再登录。点击进入下载页面 Code: 235 
[2023-06-21 14:36:20] [WARNING]: 设备信息被封禁, 请删除 device.json 后重试. 
[2023-06-21 14:36:41] [WARNING]: 虚拟设备信息不存在, 将自动生成随机设备. 
[2023-06-21 14:38:07] [WARNING]: 虚拟设备信息不存在, 将自动生成随机设备. 
[2023-06-21 14:38:18] [ERROR]: 获取T544时出现问题: Post "https://captcha.go-cqhttp.org/sdk/dandelion/energy": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 
[2023-06-21 14:38:18] [WARNING]: 登录需要滑条验证码, 请验证后重试. 
[2023-06-21 14:38:18] [WARNING]: 请选择提交滑块ticket方式: 
[2023-06-21 14:38:18] [WARNING]: 1. 自动提交 
[2023-06-21 14:38:18] [WARNING]: 2. 手动抓取提交 
[2023-06-21 14:38:18] [WARNING]: 请输入(1 - 2)： 
[2023-06-21 14:39:12] [WARNING]: 请前往该地址验证 -> https://captcha.go-cqhttp.org/captcha?id=q4zdIoPC&style=simple&aid=16&uin=2470751924&sid=4099831729245821658&cap_cd=tp_FKjvMjY29VZmvayyPRTrBRMiovrZYzRVk4E-sMXUhQWqDSZCmtA**&clientype=1&apptype=2  
[2023-06-21 14:39:31] [ERROR]: 获取T544时出现问题: Post "https://captcha.go-cqhttp.org/sdk/dandelion/energy": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 
[2023-06-21 14:39:32] [WARNING]: 登录失败: 当前QQ版本过低，请升级至最新版本后再登录。点击进入下载页面 Code: 235 
[2023-06-21 14:39:32] [WARNING]: 设备信息被封禁, 请删除 device.json 后重试. 
[2023-06-21 14:42:55] [ERROR]: 获取T544时出现问题: Post "https://captcha.go-cqhttp.org/sdk/dandelion/energy": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 
[2023-06-21 14:42:55] [WARNING]: 登录需要滑条验证码, 请验证后重试. 
[2023-06-21 14:42:55] [WARNING]: 请选择提交滑块ticket方式: 
[2023-06-21 14:42:55] [WARNING]: 1. 自动提交 
[2023-06-21 14:42:55] [WARNING]: 2. 手动抓取提交 
[2023-06-21 14:42:55] [WARNING]: 请输入(1 - 2)： 
[2023-06-21 14:42:58] [WARNING]: 请前往该地址验证 -> https://captcha.go-cqhttp.org/captcha?id=gocr13Rb&style=simple&aid=16&uin=2470751924&sid=610600774716791792&cap_cd=q8DiCqgNXQwqye9FNzwaCNbd0ltfU-j3R-BDGMrHXOFZsXysdZI5cA**&clientype=1&apptype=2  
[2023-06-21 14:44:09] [ERROR]: 获取T544时出现问题: Post "https://captcha.go-cqhttp.org/sdk/dandelion/energy": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 
[2023-06-21 14:44:10] [WARNING]: 登录失败: 请使用手Q扫描二维码或短信登录。 Code: 238 
[2023-06-21 14:44:26] [WARNING]: 账号密码未配置, 将使用二维码登录. 
[2023-06-21 14:44:26] [WARNING]: 将在 5秒 后继续. 
[2023-06-21 14:45:01] [WARNING]: 账号密码未配置, 将使用二维码登录. 
[2023-06-21 14:45:01] [WARNING]: 将在 5秒 后继续. 
[2023-06-21 14:45:34] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 14:45:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 14:45:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 14:45:56] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 14:46:25] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 14:50:02] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:14:24] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:14:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:14:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:14:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:14:44] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331672,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":27,"packet_sent":18,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:14:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:14:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:14:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:14:57] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331684,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":27,"packet_sent":18,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:14:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:15:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:15:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:15:10] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331697,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":28,"packet_sent":19,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:15:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:15:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:15:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:15:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:15:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:15:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:15:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:15:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:15:22] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331710,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":28,"packet_sent":19,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:15:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:15:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:15:25] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687331712,"self_id":2470751924,"sub_type":"normal","anonymous":null,"font":0,"group_id":734894275,"message":"[CQ:at,qq=3311599537]","message_seq":5939,"sender":{"age":0,"area":"","card":"","level":"","nickname":"LemonQu","role":"member","sex":"unknown","title":"","user_id":173887664},"user_id":173887664,"message_id":1195704479,"raw_message":"[CQ:at,qq=3311599537]"}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:15:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:15:28] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687331716,"self_id":2470751924,"sub_type":"normal","font":0,"message_seq":5940,"sender":{"age":0,"area":"","card":"","level":"","nickname":"LemonQu","role":"member","sex":"unknown","title":"","user_id":3311599537},"user_id":3311599537,"message_id":1909993212,"anonymous":null,"group_id":734894275,"message":"[CQ:at,qq=173887664] 114514","raw_message":"[CQ:at,qq=173887664] 114514"}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:15:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:15:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:15:35] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331722,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":30,"packet_sent":19,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331716}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:15:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:15:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:15:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:15:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:15:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:15:45] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687331732,"self_id":2470751924,"sub_type":"normal","anonymous":null,"font":0,"message":"[CQ:face,id=28]","message_seq":5941,"user_id":173887664,"message_id":115285610,"group_id":734894275,"raw_message":"[CQ:face,id=28]","sender":{"age":0,"area":"","card":"","level":"","nickname":"LemonQu","role":"member","sex":"unknown","title":"","user_id":173887664}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:15:48] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331735,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":32,"packet_sent":20,"packet_lost":0,"message_received":3,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331732}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:15:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:15:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:15:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:16:00] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331748,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":32,"packet_sent":20,"packet_lost":0,"message_received":3,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331732}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:16:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:16:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:16:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:16:13] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331760,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":33,"packet_sent":21,"packet_lost":0,"message_received":3,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331732}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:16:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:16:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:16:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:16:26] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331773,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":33,"packet_sent":21,"packet_lost":0,"message_received":3,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331732}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:16:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:16:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:16:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:16:39] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331786,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":34,"packet_sent":22,"packet_lost":0,"message_received":3,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331732}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:16:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:16:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:16:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:16:51] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331799,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":34,"packet_sent":22,"packet_lost":0,"message_received":3,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331732}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:16:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:16:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:17:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:17:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:17:04] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331811,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":34,"packet_sent":22,"packet_lost":0,"message_received":3,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331732}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:17:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:17:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:17:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:17:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:17:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:17:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:17:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:17:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:17:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:17:14] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687331821,"self_id":2470751924,"sub_type":"normal","font":0,"message_seq":5942,"user_id":1329913830,"message_id":-1613344816,"sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"anonymous":null,"group_id":734894275,"message":"草","raw_message":"草"}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:17:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:17:17] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331824,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":36,"packet_sent":23,"packet_lost":0,"message_received":4,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331821}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:17:17] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687331824,"self_id":2470751924,"sub_type":"normal","font":0,"message_seq":5943,"raw_message":"[CQ:at,qq=3311599537]","anonymous":null,"group_id":734894275,"message":"[CQ:at,qq=3311599537]","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"user_id":1329913830,"message_id":-388923578}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:17:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:17:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:17:21] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687331829,"self_id":2470751924,"sub_type":"normal","message":"无效森","message_seq":5944,"raw_message":"无效森","user_id":1329913830,"message_id":1991633637,"anonymous":null,"font":0,"group_id":734894275,"sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:17:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:17:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:17:30] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331837,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":38,"packet_sent":23,"packet_lost":0,"message_received":6,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":1687331829}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:17:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:17:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:18:28] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:18:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:18:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:18:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:18:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:18:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:18:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:18:48] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687331915,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":27,"packet_sent":18,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}}}
 到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:18:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:18:52] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687331919,"self_id":2470751924,"sub_type":"normal","message":"😋","raw_message":"😋","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"user_id":1329913830,"font":0,"group_id":734894275,"message_seq":5945,"message_id":28498547,"anonymous":null}
 到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:18:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:18:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5700/ 失败: Post "http://127.0.0.1:5700/": dial tcp 127.0.0.1:5700: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:20:17] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:20:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:20:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:20:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:20:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:20:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:20:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:20:37] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687332024,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":25,"packet_sent":17,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:20:39] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687332026,"self_id":2470751924,"sub_type":"normal","message_seq":5946,"raw_message":"um","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"user_id":1329913830,"message_id":-1732539447,"font":0,"group_id":734894275,"message":"um","anonymous":null}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 15:20:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:20:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:20:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:21:04] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:45:26] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:45:28] [WARNING]: HTTP 服务器使用了过时的配置格式，请更新配置文件！ 
[2023-06-21 15:45:33] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:45:38] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:45:43] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:45:48] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:45:53] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:45:58] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:03] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:08] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:13] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:18] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:23] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:28] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:33] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:38] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:43] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:46:57] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:47:05] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:47:10] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:50:13] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:50:21] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:50:26] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:50:31] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:50:36] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:50:41] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:50:46] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:51:13] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:51:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 15:51:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 15:51:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 15:52:15] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:52:22] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:52:27] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:52:29] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:52:31] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:52:32] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:52:37] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:52:42] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:52:47] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:52:52] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:52:57] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:53:02] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:53:07] [WARNING]: 上报 Event 数据到 127.0.0.1:5701 时创建请求失败: parse "127.0.0.1:5701": first path segment in URL cannot contain colon 
[2023-06-21 15:53:31] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 15:53:34] [WARNING]: 将在五秒后退出. 
[2023-06-21 16:03:15] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 16:03:23] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:04:19] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 16:04:26] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:04:31] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:04:36] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:04:41] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:04:46] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:04:51] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:04:56] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:01] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:06] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:11] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:16] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:21] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:26] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:31] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:36] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:41] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:42] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:46] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:51] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:05:56] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:06:01] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:06:06] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:06:11] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:06:16] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:06:21] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:06:26] [WARNING]: 上报 Event 数据到 127.0.0.1:5700 时创建请求失败: parse "127.0.0.1:5700": first path segment in URL cannot contain colon 
[2023-06-21 16:07:28] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 16:07:30] [WARNING]: 将在五秒后退出. 
[2023-06-21 16:07:59] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 16:08:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:08:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:08:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:08:20] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687334887,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":27,"packet_sent":18,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:08:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:08:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:08:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:10:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:10:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:10:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:14:53] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 16:15:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:15:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:20:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:20:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:20:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:20:14] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335601,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":63,"packet_sent":51,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:20:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:20:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:20:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:20:26] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335614,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":63,"packet_sent":51,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:20:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:20:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:20:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:20:39] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335626,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":64,"packet_sent":52,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:20:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:20:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:20:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:20:52] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335639,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":64,"packet_sent":52,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:20:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:20:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:21:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:21:04] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335652,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":64,"packet_sent":52,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:21:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:21:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:21:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:21:17] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335664,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":65,"packet_sent":53,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:21:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:21:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:21:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:21:30] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335677,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":65,"packet_sent":53,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:21:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:21:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:21:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:21:42] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335690,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":66,"packet_sent":54,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:21:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:21:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:21:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:21:55] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335702,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":66,"packet_sent":54,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:21:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:22:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:22:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:22:08] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335715,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":66,"packet_sent":54,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:22:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:22:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:22:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:22:20] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335728,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":68,"packet_sent":56,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:22:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:22:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:22:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:22:33] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335740,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":68,"packet_sent":56,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:22:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:22:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:22:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:22:46] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335753,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":69,"packet_sent":57,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:22:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:22:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:22:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:22:58] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335766,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":69,"packet_sent":57,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:23:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:23:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:23:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:23:11] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335778,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":70,"packet_sent":58,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:23:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:23:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:23:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:23:24] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335791,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":70,"packet_sent":58,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:23:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:23:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:23:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:23:37] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687335804,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":70,"packet_sent":58,"packet_lost":0,"message_received":4,"message_sent":3,"disconnect_times":0,"lost_times":0,"last_message_time":1687335473}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:23:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:23:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:23:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:25:26] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 16:27:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:27:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:27:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:27:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:27:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:27:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:27:56] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336063,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":63,"packet_sent":41,"packet_lost":0,"message_received":8,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336055}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:27:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:27:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:27:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:27:59] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687336067,"self_id":2470751924,"sub_type":"normal","user_id":1329913830,"group_id":734894275,"message":"这玩意不能多玩","message_seq":5985,"sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"message_id":-1392011905,"anonymous":null,"font":0,"raw_message":"这玩意不能多玩"}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:28:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:28:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:28:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:28:04] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687336072,"self_id":2470751924,"sub_type":"normal","anonymous":null,"message":"先关了","raw_message":"先关了","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"message_id":873383109,"font":0,"group_id":734894275,"message_seq":5986,"user_id":1329913830}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:28:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:28:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:28:08] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336076,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":65,"packet_sent":41,"packet_lost":0,"message_received":10,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336072}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:28:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:28:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:28:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:28:13] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687336080,"self_id":2470751924,"sub_type":"normal","font":0,"group_id":734894275,"message_id":1124725843,"raw_message":"不过端午节我要出门","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"user_id":1329913830,"anonymous":null,"message":"不过端午节我要出门","message_seq":5987}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:28:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:28:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:28:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:28:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:28:20] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687336088,"self_id":2470751924,"sub_type":"normal","anonymous":null,"message_seq":5988,"raw_message":"可能无法继续写了（悲","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"user_id":1329913830,"message_id":-742987326,"font":0,"group_id":734894275,"message":"可能无法继续写了（悲"}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:28:21] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336088,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":68,"packet_sent":42,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:28:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:28:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:28:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:28:34] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336101,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":68,"packet_sent":42,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:28:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:28:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:28:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:28:46] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336114,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":69,"packet_sent":43,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:28:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:28:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:28:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:28:59] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336126,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":69,"packet_sent":43,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:29:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:29:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:29:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:29:12] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336139,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":45,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:29:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:29:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:29:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:29:24] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336152,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":45,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:29:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:29:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:29:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:29:37] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336164,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":45,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:29:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:29:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:29:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:29:50] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336177,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":72,"packet_sent":46,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:29:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:29:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:29:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:30:02] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336190,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":72,"packet_sent":46,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:30:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:30:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:30:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:30:15] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336202,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":73,"packet_sent":47,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:30:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:30:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:30:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:30:28] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336215,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":73,"packet_sent":47,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:30:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:30:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:30:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:30:40] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336228,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":74,"packet_sent":48,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:30:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:30:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:30:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:30:53] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336240,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":74,"packet_sent":48,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:30:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:30:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:31:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:31:06] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336253,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":74,"packet_sent":48,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:31:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:31:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:31:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:31:18] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336266,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":76,"packet_sent":49,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:31:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:31:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:31:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:31:31] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336278,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":76,"packet_sent":49,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:31:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:31:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:31:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:31:44] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336291,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":77,"packet_sent":50,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:31:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:31:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:31:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:31:56] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336304,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":77,"packet_sent":50,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:31:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:32:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:32:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:32:09] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336316,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":77,"packet_sent":50,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:32:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:32:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:32:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:32:22] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336329,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":78,"packet_sent":51,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:32:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:32:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:32:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:32:35] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336342,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":78,"packet_sent":51,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:32:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:32:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:32:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:32:47] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336355,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":80,"packet_sent":53,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:32:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:32:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:32:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:33:00] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336367,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":80,"packet_sent":53,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:33:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:33:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:33:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:33:13] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336380,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":81,"packet_sent":54,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:33:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:33:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:33:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:33:25] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336393,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":81,"packet_sent":54,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:33:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 16:33:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 16:33:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 16:33:38] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687336405,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":81,"packet_sent":54,"packet_lost":0,"message_received":12,"message_sent":7,"disconnect_times":0,"lost_times":0,"last_message_time":1687336088}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 16:38:30] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 20:30:11] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 20:30:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 20:30:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 20:30:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 20:59:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 20:59:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 20:59:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 20:59:31] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687352358,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":104,"packet_sent":86,"packet_lost":0,"message_received":7,"message_sent":2,"disconnect_times":0,"lost_times":0,"last_message_time":1687352332}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 20:59:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 20:59:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:00:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:00:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:00:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:00:56] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687352443,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":114,"packet_sent":90,"packet_lost":0,"message_received":11,"message_sent":4,"disconnect_times":0,"lost_times":0,"last_message_time":1687352426}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:00:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:01:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:01:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:01:09] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687352456,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":115,"packet_sent":91,"packet_lost":0,"message_received":11,"message_sent":4,"disconnect_times":0,"lost_times":0,"last_message_time":1687352426}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:01:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:01:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:01:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:01:21] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687352469,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":115,"packet_sent":91,"packet_lost":0,"message_received":11,"message_sent":4,"disconnect_times":0,"lost_times":0,"last_message_time":1687352426}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:01:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:01:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:01:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:01:34] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687352481,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":116,"packet_sent":92,"packet_lost":0,"message_received":11,"message_sent":4,"disconnect_times":0,"lost_times":0,"last_message_time":1687352426}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:01:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:01:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:01:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:01:47] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687352494,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":116,"packet_sent":92,"packet_lost":0,"message_received":11,"message_sent":4,"disconnect_times":0,"lost_times":0,"last_message_time":1687352426}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:01:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:15:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2023-06-21 21:15:16] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:15:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2023-06-21 21:15:22] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:15:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2023-06-21 21:15:29] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:15:35] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687353311,"self_id":2470751924,"sub_type":"normal","anonymous":null,"font":0,"message_seq":6072,"raw_message":"[CQ:at,qq=2470751924] 查询数据 id 1","message_id":-1449162638,"group_id":734894275,"message":"[CQ:at,qq=2470751924] 查询数据 id 1","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"user_id":1329913830}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上限 
[2023-06-21 21:15:35] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:16:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2023-06-21 21:16:26] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:16:31] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:16:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2023-06-21 21:16:32] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:16:38] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:16:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2023-06-21 21:16:39] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:16:44] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:16:45] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687353381,"self_id":2470751924,"sub_type":"normal","font":0,"group_id":734894275,"message":"[CQ:at,qq=2470751924] 查询数据 id 1","message_id":-1073089135,"anonymous":null,"message_seq":6080,"raw_message":"[CQ:at,qq=2470751924] 查询数据 id 1","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"user_id":1329913830}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上限 
[2023-06-21 21:16:45] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:16:51] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:17:45] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 21:18:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2023-06-21 21:18:30] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:18:35] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:18:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2023-06-21 21:18:36] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:18:41] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:18:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2023-06-21 21:18:43] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:18:48] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:18:49] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687353505,"self_id":2470751924,"sub_type":"normal","anonymous":null,"group_id":734894275,"message_seq":6087,"user_id":1329913830,"message_id":1584226354,"font":0,"message":"[CQ:at,qq=2470751924] 查询数据 id 1","raw_message":"[CQ:at,qq=2470751924] 查询数据 id 1","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上限 
[2023-06-21 21:18:49] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:18:54] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:19:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2023-06-21 21:19:05] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:19:11] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:19:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2023-06-21 21:19:12] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:19:17] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:19:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2023-06-21 21:19:19] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:19:24] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:19:25] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687353541,"self_id":2470751924,"sub_type":"normal","font":0,"group_id":734894275,"message_seq":6092,"message_id":924624380,"anonymous":null,"message":"[CQ:at,qq=2470751924] time","raw_message":"[CQ:at,qq=2470751924] time","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"user_id":1329913830}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上限 
[2023-06-21 21:19:25] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:19:30] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:19:47] [FATAL]: open cache image db failed: The process cannot access the file because it is being used by another process. 
[2023-06-21 21:19:49] [FATAL]: open cache image db failed: The process cannot access the file because it is being used by another process. 
[2023-06-21 21:19:54] [WARNING]: 虚拟设备信息不存在, 将自动生成随机设备. 
[2023-06-21 21:19:54] [WARNING]: 当前协议不支持二维码登录, 请配置账号密码登录. 
[2023-06-21 21:19:56] [WARNING]: 当前协议不支持二维码登录, 请配置账号密码登录. 
[2023-06-21 21:19:59] [WARNING]: 当前协议不支持二维码登录, 请配置账号密码登录. 
[2023-06-21 21:20:22] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 21:20:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2023-06-21 21:20:35] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:20:40] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:20:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2023-06-21 21:20:41] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:20:46] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:20:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2023-06-21 21:20:48] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:20:53] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687353630,"self_id":2470751924,"sub_type":"normal","message":"[CQ:at,qq=2470751924] time","user_id":1329913830,"sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"message_id":-674673438,"anonymous":null,"font":0,"group_id":734894275,"message_seq":6098,"raw_message":"[CQ:at,qq=2470751924] time"}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": read tcp 127.0.0.1:59878->127.0.0.1:5701: wsarecv: An existing connection was forcibly closed by the remote host. 停止上报：已达重试上限 
[2023-06-21 21:20:53] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:20:54] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:22:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2023-06-21 21:22:14] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:22:19] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:22:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2023-06-21 21:22:20] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:22:25] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:22:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2023-06-21 21:22:27] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:22:32] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:22:33] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687353729,"self_id":2470751924,"sub_type":"normal","raw_message":"[CQ:at,qq=2470751924] time","message_id":243257783,"anonymous":null,"font":0,"group_id":734894275,"message":"[CQ:at,qq=2470751924] time","message_seq":6104,"sender":{"age":0,"area":"","card":"","level":"","nickname":"LemonQu","role":"member","sex":"unknown","title":"","user_id":173887664},"user_id":173887664}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上限 
[2023-06-21 21:22:33] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:22:38] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:24:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2023-06-21 21:24:57] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:28:49] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 21:30:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2023-06-21 21:30:52] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:30:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2023-06-21 21:30:59] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:31:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2023-06-21 21:31:05] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:31:12] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687354248,"self_id":2470751924,"sub_type":"normal","user_id":1329913830,"font":0,"group_id":734894275,"message_seq":6163,"raw_message":"[CQ:at,qq=2470751924] time","sender":{"age":0,"area":"","card":"","level":"","nickname":"MorishimaSann","role":"owner","sex":"unknown","title":"","user_id":1329913830},"anonymous":null,"message":"[CQ:at,qq=2470751924] time","message_id":-968760430}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上限 
[2023-06-21 21:31:12] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 21:33:01] [ERROR]: Protocol -> unexpected disconnect: read tcp 192.168.1.3:62009->36.155.205.230:8080: wsarecv: An established connection was aborted by the software in your host machine. 
[2023-06-21 21:33:08] [ERROR]: Protocol -> unexpected disconnect: read tcp 192.168.31.127:51053->36.155.205.252:8080: wsarecv: An established connection was aborted by the software in your host machine. 
[2023-06-21 21:33:45] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 21:34:55] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 21:35:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:36:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:36:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:36:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:36:15] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687354562,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":28,"packet_sent":19,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:36:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:39:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:39:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:39:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:39:50] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687354777,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":36,"packet_sent":27,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:39:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:39:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:39:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:40:02] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687354790,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":36,"packet_sent":27,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:40:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:40:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:40:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:40:15] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687354802,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":37,"packet_sent":28,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:40:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:40:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:40:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:40:28] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687354815,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":37,"packet_sent":28,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:40:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:40:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:41:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:41:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:41:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:41:20] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687354867,"self_id":2470751924,"interval":5000,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":39,"packet_sent":30,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}}}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:41:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:41:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:41:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:41:32] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687354880,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":39,"packet_sent":30,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:41:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:41:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:41:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:41:45] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687354892,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":40,"packet_sent":31,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:41:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:41:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:41:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-06-21 21:41:58] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1687354905,"self_id":2470751924,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":40,"packet_sent":31,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":0,"lost_times":0,"last_message_time":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-06-21 21:42:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:43:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-06-21 21:43:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-06-21 21:44:51] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 21:50:18] [WARNING]: 当前协议不支持二维码登录, 请配置账号密码登录. 
[2023-06-21 21:50:28] [FATAL]: open cache image db failed: The process cannot access the file because it is being used by another process. 
[2023-06-21 21:50:30] [FATAL]: open cache image db failed: The process cannot access the file because it is being used by another process. 
[2023-06-21 21:50:31] [FATAL]: open cache image db failed: The process cannot access the file because it is being used by another process. 
[2023-06-21 21:50:51] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 21:55:28] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 22:01:10] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 22:08:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2023-06-21 22:08:16] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 22:08:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2023-06-21 22:08:22] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 22:08:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2023-06-21 22:08:29] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 22:08:35] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1687356491,"self_id":2470751924,"sub_type":"normal","message":"[CQ:at,qq=2470751924] 欢迎来到我们的天气之子吧官群小分队[CQ:image,file=555cff4a983af967024868a188415744.image,subType=0,url=https://gchat.qpic.cn/gchatpic_new/2854196310/719772033-3093978372-555CFF4A983AF967024868A188415744/0?term=2\u0026amp;is_origin=0]\n——来自群管理员（1329913830）的编辑","raw_message":"[CQ:at,qq=2470751924] 欢迎来到我们的天气之子吧官群小分队[CQ:image,file=555cff4a983af967024868a188415744.image,subType=0,url=https://gchat.qpic.cn/gchatpic_new/2854196310/719772033-3093978372-555CFF4A983AF967024868A188415744/0?term=2\u0026amp;is_origin=0]\n——来自群管理员（1329913830）的编辑","sender":{"age":0,"area":"","card":"","level":"","nickname":"Q群管家","role":"member","sex":"unknown","title":"","user_id":2854196310},"user_id":2854196310,"message_id":257194698,"anonymous":null,"font":0,"group_id":719772033,"message_seq":67799}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 停止上报：已达重试上限 
[2023-06-21 22:08:35] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2023-06-21 22:09:48] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-06-21 22:10:19] [WARNING]: 群消息发送失败: 账号可能被风控. 
